pkg/liqo-controller-manager/namespace-controller/manage_remote_namespaces.go

```diff
@@ -42,16 +44,23 @@ func (r *NamespaceReconciler) addDesiredMapping(n *corev1.Namespace, remoteName
 	}
 
 	if oldValue, ok := nm.Spec.DesiredMapping[n.GetName()]; ok {
-		// if entries are already here, but mappingLabel has a different value from the previous one, we force again the old value.
-		// Common user cannot change remote namespace name while the namespace is offloaded onto remote clusters
+		// if entry is already here, but mappingLabel has a different value from the previous one, we force again the old value.
+		// User's cannot change remote namespace name while the namespace is offloaded onto remote clusters
 		if oldValue != remoteName {
 			n.GetLabels()[mappingLabel] = oldValue
+			if n.GetAnnotations() == nil {
+				n.Annotations = map[string]string{}
+			}
+			n.GetAnnotations()[mappingAnnotationRenaming] = fmt.Sprintf("You cannot change the value of label [%s] because namespace's offload has already started with name [%s]. Please read our documentation for more info [%s]",
```

Comment:
```suggestion
			n.GetAnnotations()[mappingAnnotationRenaming] = fmt.Sprintf("You cannot change the value of label [%s] because the namespace is already offloaded with name [%s]. Please read our documentation for more info [%s]",
```

Reply:
Thank you